src/views/Home.jsx

```diff
@@ -1,6 +1,31 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
+
 export function Home({ data, setListPath }) {
+	const [listName, setListName] = useState('');
+	const [error, setError] = useState('');
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+	const navigate = useNavigate();
+
+	async function handleSubmit(e) {
+		e.preventDefault();
+
```

Comment:
Looks good so far! During testing, I couldn't submit an emtpy string, but if I added a couple spaces, the list was created